project/devs.sls

```diff
@@ -3,7 +3,7 @@ include:
 
 {% if 'users' in pillar and pillar['users'] %}
 {% for user, args in pillar['users'].iteritems() %}
-{{ user }}:
+dev_{{ user }}:
```

Comment:
I've changed it and it still works.  Maybe this is because you can either refer to a state by its parent key (dev_{{ user }}) or by its "name:" value ({{ user }}).